syncer/test/sotrage/storage.go

```diff
@@ -18,24 +18,19 @@
 package storage
```

Comment:
Actually, we should move folder of mock tests into syncer/pkg/mocks, include sycner/test/dcmock and syncer/test/servicecenter.
I am going to do this.
So please mv syncer/test/storage to syncer/pkg/mocks.

Reply:
You can reference to https://github.com/etcd-io/etcd/tree/master/pkg/mock.
To syncer, folder maybe like following,
-- syncer
    -- pkg
       -- mock
           -- mockservicecenter
           -- mockstorage
           -- mockxxx

Reply:
I will do it,thanks for pointing

Reply:
@zenlint Edit completed, please view.